presto-product-tests/bin/run_on_docker.sh

```diff
@@ -70,25 +70,46 @@ function stop_application_runner_containers() {
   done
 }
 
+function stop_all_containers() {
+  local ENVIRONMENT
+  for ENVIRONMENT in $(getAvailableEnvironments)
```

Comment:
Maybe i'm wrong, but as i know `local` is visible only within the function where it is declared. If i'm right the `environment_docker_compose` in `stop_docker_compose_containers` will use global `ENVIRONMENT` variable and will stop the user selected environment for the 4 times. I'm not a big bash specialist, so i could be wrong. But you better check it.